src/components/common/PageFramework/PageFramework.module.scss

```diff
@@ -1,6 +1,15 @@
 @import '../../../scss/variables';
 
 .main {
+  background: var(--a-gray-100);
+  min-height: 55vh;
```

Comment:
Trenger vi både `min-height: 55vh;` i `.main `og i `#root`?
Kan den kanskje fjernes fra `#root `i såfall?

Reply:
Bør være der for initial render før javascript lastes